trussworks/react-uswds
# feat: Deprecate the Modal component

> :warning: **Per #787 the modal component is to be relocated to a Truss-specific repo**: The modal component will always be available in the git history, but after merging this PR, that will be the _only_ place to find it.

# Summary

The Modal Component is no a part of `uswds` and is therefore being removed from the library.

As this is a breaking change, it is targeting the V2 release f `react-uswds`

## Related Issues or PRs

closes https://github.com/trussworks/react-uswds/issues/787

## How To Test

1. Checkout this branch
```
git fetch
git checkout bl-deprecate-modal-breaking-787
```
2. Verify all references to `Modal` are removed. 
3. Example App runs (Modal **and** redux have been removed from the example app)
